types/fullcalendar__vue/index.d.ts

```diff
@@ -0,0 +1,193 @@
+    defaultRangeSeparator: object;
+    selectConstraint: object;
+    selectOverlap: object;
+    selectAllow: object;
+    editable: object;
+    eventStartEditable: object;
+    eventDurationEditable: object;
+    eventConstraint: object;
+    eventOverlap: object;
+    eventAllow: object;
+    eventClassName: object;
+    eventClassNames: object;
+    eventBackgroundColor: object;
+    eventBorderColor: object;
+    eventTextColor: object;
+    eventColor: object;
+    events: object;
+    eventSources: object;
+    allDayDefault: object;
+    startParam: object;
+    endParam: object;
+    lazyFetching: object;
+    nextDayThreshold: object;
+    eventOrder: object;
+    rerenderDelay: object;
+    dragRevertDuration: object;
+    dragScroll: object;
+    longPressDelay: object;
+    eventLongPressDelay: object;
+    droppable: object;
+    dropAccept: object;
+    eventDataTransform: object;
+    allDayMaintainDuration: object;
+    eventResizableFromStart: object;
+    timeGridEventMinHeight: object;
+    allDayHtml: object;
+    eventDragMinDistance: object;
+    eventResourceEditable: object;
+    eventSourceFailure: object;
+    eventSourceSuccess: object;
+    forceEventDuration: object;
+    progressiveEventRendering: object;
+    selectLongPressDelay: object;
+    selectMinDistance: object;
+    timeZoneParam: object;
+    titleRangeSeparator: object;
+    // compound OptionsInput...
+    buttonText: object;
+    views: object;
+    plugins: object;
+    // scheduler...
+    schedulerLicenseKey: object;
+    resources: object;
+    resourceLabelText: object;
+    resourceOrder: object;
+    filterResourcesWithEvents: object;
+    resourceText: object;
+    resourceGroupField: object;
+    resourceGroupText: object;
+    resourceAreaWidth: object;
+    resourceColumns: object;
+    resourcesInitiallyExpanded: object;
+    slotWidth: object;
+    datesAboveResources: object;
+    googleCalendarApiKey: object;
+    refetchResourcesOnNavigate: object;
+    // used to be emissions but are now props...
+    datesRender: object;
+    datesDestroy: object;
+    dayRender: object;
+    eventRender: object;
+    eventDestroy: object;
+    viewSkeletonRender: object;
+    viewSkeletonDestroy: object;
+    resourceRender: object;
+}
+
+interface PropIsDeep {
+    header: boolean;
+    footer: boolean;
+    events: boolean;
+    eventSources: boolean;
+    resources: boolean;
+}
+
+interface EmissionUseProp {
+    datesRender: boolean;
+    datesDestroy: boolean;
+    dayRender: boolean;
+    eventRender: boolean;
+    eventDestroy: boolean;
+    viewSkeletonRender: boolean;
+    viewSkeletonDestroy: boolean;
+    resourceRender: boolean;
+}
+
+interface CalendarComponent extends Vue {
+    buildCalendarOptions: () => void;
+    recordDirtyOption: (optionName: any, newVal: any) => void;
+    renderDirty: () => void;
+    getApi: () => any;
+}
+
+export type FullCalendarComponent = CalendarComponent;
+export const PROP_DEFS: PropsDef;
```

Comment:
These constants are not exported by the library

Reply:
Hi thanks for reviewing this soo quickly. I'll remove them. I wasn't sure if they should be included because they are in https://github.com/fullcalendar/fullcalendar-vue/blob/master/src/fullcalendar-options.js. Now you mention it I probably should be basing this on the bundled npm package. I will remove and update.